moduleframework/tests/static/dockerfile_lint.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import print_function
+
+import os
+from avocado import Test
+from moduleframework import module_framework
```

Comment:
remove this line and remove ``module_framework.AvocadoTest`` from class parent

Reply:
It is not possible, because we have in AvocadoTest function asser_to_warn.